charts/atlantis/values.yaml

```
@@ -508,10 +508,10 @@ servicemonitor:
     externalSecret:
       # authentication based on an external secret
       enabled: false
-      name: atlantis-env
-      keys:
-        username: USERNAME
-        password: ATLANTIS_WEB_PASSWORD
+      # name: atlantis-env
+      # keys:
+        # username: USERNAME
+        # password: ATLANTIS_WEB_PASSWORD
```

nit: keep the comment blocks aligned, similar to existing ones:
```suggestion
      # name: atlantis-env
      # keys:
      #   username: USERNAME
      #   password: ATLANTIS_WEB_PASSWORD
```